components/meter/meter-circle.js

```diff
@@ -10,11 +10,16 @@ import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
  */
 class MeterCircle extends MeterMixin(RtlMixin(LitElement)) {
 	static get styles() {
-		return [ bodyStandardStyles, css`
+		return [ bodySmallStyles, bodyStandardStyles, css`
 		:host {
 			display: inline-block;
 			width: 2.4rem;
 		}
+		.d2l-meter-circle {
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+		}
```

Comment:
RE: refactoring the styles, depending on how much can be consolidated I think it could make sense?